setup/start.sh

```diff
@@ -147,7 +64,8 @@ source setup/zpush.sh
 source setup/management.sh
 
 # Ping the management daemon to write the DNS and nginx configuration files.
-while [ ! -f /var/lib/mailinabox/api.key ]; do
+until nc -z -w 4 localhost 10222
```

Comment:
If you change this, you also have to remove these lines in `management.sh`:

     . . . Remove the api key file first so that start.sh
     # can wait for it to be created to know that the management
     # server is ready.
     rm -f /var/lib/mailinabox/api.key

What was the reason for the change? Might want to put into a separate commit.